rackcity/utils/asset_utils.py

```diff
@@ -471,6 +471,8 @@ def save_all_field_data_cp(data, asset, change_plan, create_asset_cp):
             value = ITModel.objects.get(id=data[field])
         elif field == "rack" and data["rack"]:
             value = Rack.objects.get(id=data[field])
+        elif field == "offline_storage_site" and data["offline_storage_site"]:
+            value = Site.objects.get(id=data[field])
```

Comment:
this fix was added to `save_all_field_data_live` way back when but missed this method